### PR TITLE
Fixed Boolean test on mergeable_features

### PR DESCRIPTION
**Pull request by RuggeroFabbiano**
The error is just formal, I confirm no change in the results

### CODE REVIEW

**Comment by pgrandinetti** [comment]
@RuggeroFabbiano 
Thanks. Can you explain the fix in plain English?
Also, it seems that the output (and counter) of the cells didn't change downstream the modified one. Is that correct, and did you re-run all of them?

**Comment by RuggeroFabbiano** [comment]
> @RuggeroFabbiano
> Thanks. Can you explain the fix in plain English?
> Also, it seems that the output (and counter) of the cells didn't change downstream the modified one. Is that correct, and did you re-run all of them?

I realised I made a mistake in computing the Boolean `mergeable_features` to check if vehicles and pedestrian features were completely independent: basically I was checking if there was any row with independent values, instead of checking if all of them were so. Didn't re-run the subsequent cells because the output of the only one I modified didn't change (all couple of features are indeed independent, even if I was checking it wrong).
